src/lib/api.js

```diff
@@ -0,0 +1,114 @@
+import axios from 'axios';
+import pickBy from 'lodash/pickBy';
+
+const api = {
```

Comment:
This looks more like a more efficient way to structure a util file. we should probably do this to `utils` at some point as well.

Reply:
Agreed.